MERP/merp_2nd_ed_sheet.html

```diff
@@ -2632,10 +2632,11 @@ <h2>Skills</h2>
                         <label title="The available number of power points is based on stats (IG or IT) and level.">Power Points:</label>
                     </div>
                     <div class="table-cell">
-                        <input type="hidden" name="attr_power_points_base" value="0"/>
-                        <input type="hidden" name="attr_effective_pp_multiplier" value="(((@{power_point_multiplier} + 1) + abs(@{power_point_multiplier} - 1))/2)"/>
+                        <input type="hidden" name="attr_power_points_base" value="0" />
+                        <input type="hidden" name="attr_effective_pp_multiplier" value="(((@{power_point_multiplier} + 1) + abs(@{power_point_multiplier} - 1))/2)" />
                         <input type="number" name="attr_power_points" value="0" />/
-                        <input type="number" name="attr_total_power_points" value="(@{power_points_base} * @{level} * @{effective_pp_multiplier})" disabled />
+                        <input type="hidden" name="attr_total_power_points" value="(@{power_points_base} * @{level} * @{effective_pp_multiplier})" disabled />
+                        <input type="number" name="attr_power_points_max" value="(@{power_points_base} * @{level} * @{effective_pp_multiplier})" disabled />
```

Comment:
Hid attr_total_power_points, but left it in attributes in case anyone's using it for macros.
Added power_points_max so token bars would work if linked to power_points attribute.